app/src/main/java/protect/card_locker/LoyaltyCardEditActivity.java

```diff
@@ -880,7 +888,13 @@ public void onResume() {
             }
         }
 
-        if (cardId != null) {
+        CharSequence cardFieldTextViewError = cardIdFieldView.getError();
+
+        if (cardFieldTextViewError != null) {
+            Snackbar.make(cardIdFieldView,
+                    cardFieldTextViewError.toString(),
+                    Snackbar.LENGTH_LONG).show();
+        } else if (cardId != null) {
```

Comment:
I think it's this part of code that's causing it, but there's definitely something broken now.

1. Start Catima
2. Click +
3. Click Manually enter ID
4. Enter "abc" in Card ID
5. Click "Aztec"
6. Card ID field is empty and 2 errors pop up

![image](https://user-images.githubusercontent.com/1885159/216816547-70594972-05ea-4e1a-b7db-e4c026950e6b.png)

Reply:
Removed the logic above and now only one error shows up (the one regarding the name field being empty).